diff --git configure/RELEASE configure/RELEASE
index 642415d..7b8a206 100644
--- configure/RELEASE
+++ configure/RELEASE
@@ -23,3 +23,8 @@ EPICS_BASE=/home/oxygen/MOONEY/epics/bazaar/base-3.14
 #Capfast users may need the following definitions
 #CAPFAST_TEMPLATES=
 #SCH2EDIF_PATH=
+
+
+# The definitions shown below can also be placed in an untracked RELEASE.local
+-include $(TOP)/../RELEASE.local
+-include $(TOP)/configure/RELEASE.local
diff --git tests/Makefile tests/Makefile
index 7230c2d..e06cf81 100644
--- tests/Makefile
+++ tests/Makefile
@@ -21,6 +21,14 @@ calcTestHarness_SRCS += $(testHarness_SRCS)
 TESTSPEC_vxWorks = calcTestHarness.munch; runTests
 
 PROD_LIBS += calc
+
+ifdef SSCAN
+PROD_LIBS += sscan
+endif
+ifdef SNCSEQ
+PROD_LIBS += seq
+endif
+
 PROD_LIBS += $(EPICS_BASE_IOC_LIBS)
 
 TESTSCRIPTS_HOST += $(TESTS:%=%.t)
